verified_model/Site/North/reference_point_latitude.json

```diff
@@ -0,0 +1,75 @@
+{
+    "value": 28.762158,
```

Comment:
Are these enough digits? Konrad uses more and in your UTM files it is not listed (but I remember it was an issue at some point).

Reply:
We do not use and should not use anywhere long/lat for coordinate transformations. As you say, it is a pain getting enough digits and we kind of agreed in our long discussions on the transformation topic in Dec/Nov that for array element positions, we use UTM -> ground.

Lat/Long is need for e.g., tools for the geomagnetic field or for atmospheric parameters. So the accuracy is enough here.

Should I add an explicit comment in the schema that we should use long/lat for coordinate transformations?

Reply:
No, it is fine. It came to my mind just because of the issues we had a few months ago. In the future it will be forgotten and the values are OK as is.